tests/abs-float16.ispc

```diff
@@ -1,6 +1,5 @@
 #include "../test_static.isph"
-// rule: skip on arch=x86
-// rule: skip on arch=x86-64
+
```

Comment:
This PR doesn't have changes to make it work on x86, so we should probably not relax this restriction yet.